source/_integrations/zha.markdown

```diff
@@ -380,27 +382,36 @@ This service disables a lock code on a Zigbee lock.
 
 ## Adding devices
 
-To add a new device:
+Tip! Regardless if your timeline is now before or after you started adding devices to the ZHA integration, it is highly recommended that you read through the two segments under the troubleshooting section below about "*Best practices to avoid pairing/connection difficulties*" and "*Zigbee interference avoidance and network range/coverage optimization*" for general prerequisite knowledge and advice on how to achieve the best possible Zigbee network in your environment.
+
+**To add a new Zigbee device:**
 
 1. Go to the **Integrations** panel, find the **Zigbee Home Automation** integration that was added by the configuration steps above, and select **Configure**.
 1. Click on the plus button at the bottom right corner to start a scan for new devices.
-1. Reset your Zigbee devices according to the device instructions provided by the manufacturer (e.g., turn on/off lights up to 10 times, switches usually have a reset button/pin). It might take a few seconds for the devices to appear. You can click on **Show logs** for more verbose output.
+1. Reset your Zigbee devices to fectory default settings according to the device instructions provided by the manufacturer (e.g., turn on/off lights up to 10 times, switches usually have a reset button/pin). It might take a few seconds for the devices to appear. You can click on **Show logs** for more verbose output.
 1. Once the device is found, it will appear on that page and will be automatically added to your devices. You can optionally change its name and add it to an area (you can change this later). You can search again to add another device, or you can go back to the list of added devices.
 
-### Using router devices
+### Using router devices to add more devices
+
+You can normally assume that most mains-powered devices, e.g., many always-powered wall plugs or lightbulbs in your Zigbee network will automatically act as a Zigbee Router Device (sometimes also referred to as a Zigbee "signal repeater" or "range extended"). 
```

Comment:
```suggestion
Most mains-powered devices, e.g., many always-powered wall plugs or light bulbs in your Zigbee network will automatically act as a Zigbee router device (sometimes also referred to as a Zigbee "signal repeater" or "range extender"). 
```

Reply:
Would it make sense to give a bit of high-level information somewhere on the Zigbee device types? Something like: There are three types of Zigbee devices: end devices, routers, and coordinators. Most mains-powered devices, e.g. , many ....